products/amazon-rds-mariadb.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Amazon RDS for MariaDB
+category: service
+tags: amazon database
+iconSlug: amazonrds
+permalink: /amazon-rds-mariadb
+releasePolicyLink: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MariaDB.Concepts.VersionMgmt.html
+releaseDateColumn: true
+
+auto:
+  methods:
+  -   custom: rds
+  -   release_table: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MariaDB.Concepts.VersionMgmt.html
+      selector: "#rds-mariadb-version-release-calendar table"
+      fields:
+        releaseCycle:
+          column: "MariaDB major version"
+          regex: '^MariaDB\s+(?P<value>\d+\.\d+).*$'
+        eol: "RDS end of standard support date"
+
+releases:
+-   releaseCycle: "10.11"
+    releaseDate: 2023-08-21
+    eol: 2028-02-29
+    latest: "10.11.8"
+    latestReleaseDate: 2024-06-18
+
+-   releaseCycle: "10.6"
+    releaseDate: 2022-02-03
+    eol: 2026-07-31
+    latest: "10.6.18"
+    latestReleaseDate: 2024-06-14
+
+-   releaseCycle: "10.5"
+    releaseDate: 2021-01-21
+    eol: 2025-06-30
+    latest: "10.5.25"
+    latestReleaseDate: 2024-06-14
+
+-   releaseCycle: "10.4"
+    releaseDate: 2020-04-06
+    eol: 2025-02-28
+    latest: "10.4.34"
+    latestReleaseDate: 2024-06-14
+---
```

Comment:
It could be interesting to document a few more releases using previous versions of the 'MariaDB on Amazon RDS versions' page (https://web.archive.org/web/20230401000000*/https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MariaDB.Concepts.VersionMgmt.html). WDYT ?

Reply:
Hello @marcwrobel I found all the information about the `10.3` release so I added it. I'm struggling to find a full set of information for the previous one, so I propose to stop at this version (new instances of this version cannot be created since August 23, 2023.